### ISSUE TITLE
Is  that any way to set the node priority  as it do in the admin api?

### ISSUE THREAD

**Issue by xianshun163** [apache/apisix-ingress-controller#935]
### Issue description

the function is  like  the nginx  config:
upstream server_tomcat1 {
server 127.0.0.1:9001 weight=1 max_fails=5  fail_timeout=60s;
server 127.0.0.1:9010 weight=1  max_fails=5  fail_timeout=60s backup;
}

And  the apisix  admin api doc  like below:
https://apisix.apache.org/docs/apisix/admin-api

Note:
Each node can be configured with a priority. A node with low priority will only be used when all the nodes with higher priority have been tried or are unavailable.

As the default priority is 0, nodes with negative priority can be configured as a backup.

For example:

{
  "uri": "/hello",
  "upstream": {
    "type": "roundrobin",
    "nodes": [
      { "host": "127.0.0.1", "port": 1980, "weight": 2000 },
      { "host": "127.0.0.1", "port": 1981, "weight": 1, "priority": -1 }
    ],
    "checks": {
      "active": {
        "http_path": "/status",
        "healthy": {
          "interval": 1,
          "successes": 1
        },
        "unhealthy": {
          "interval": 1,
          "http_failures": 1
        }
      }
    }
  }
}
Node 127.0.0.2 will be used only after 127.0.0.1 is tried or unavailable. It can therefore act as a backup for the node 127.0.0.1.

### Environment

- your apisix-ingress-controller version (output of apisix-ingress-controller version --long):
- your Kubernetes cluster version (output of kubectl version):
- if you run apisix-ingress-controller in Bare-metal environment, also show your OS version (uname -a):
v1.4.0

**Comment by tao12345666333** [comment]
In fact in Kubernetes, our backends are Pods, and they don't actually make any difference.

But we do have a way to do what you need (if you really need it)
You can use subset to achieve this. Please refer to https://github.com/apache/apisix-ingress-controller/blob/master/test/e2e/features/subset.go

**Comment by xianshun163** [comment]
ok， I will try it later  thanks.

**Comment by ryan4yin** [comment]
> 

@tao12345666333 APISIX's subsets is very similar to Istio, it's a nice feature, but I wondering how to make a service's subset as a backup. 

According to the docs https://github.com/apache/apisix-ingress-controller/blob/master/docs/en/latest/references/apisix_route_v2.md, it seems that only subset's weight can be modified via `http[].backends[].weight`, and there is no way to set subset's `priority` to `-1`.

**Comment by tao12345666333** [comment]
@ryan4yin refer to https://github.com/apache/apisix-ingress-controller/tree/master/test/e2e/suite-features

The subset case
